tethysext/atcore/tests/integrated_tests/services/resource_workflows/helpers_tests.py

```diff
@@ -33,6 +33,20 @@ def test_set_step_status(self):
     @mock.patch('argparse._sys')
     def test_parse_workflow_step_args(self, mock_sys):
         mock_sys.argv = ['prog']
-        ret = helpers.parse_workflow_step_args()
+        ret, _ = helpers.parse_workflow_step_args()
 
         self.assertIsInstance(ret, Namespace)
+
+    @mock.patch('argparse._sys')
+    def test_parse_workflow_step_args_with_extra(self, mock_sys):
+        mock_sys.argv = ['prog']  # No extra argument
+        ret, extra_args = helpers.parse_workflow_step_args()
+
+        self.assertIsInstance(ret, Namespace)
+        self.assertListEqual(extra_args, [])
+
+        mock_sys.argv = ['prog', '--extra_arg']  # Extra argument
```

Comment:
Has this been tested with multiple extra arguments?

Reply:
Tests will be updated.